File: 0242-valid-anagram/0242-valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        if len(s) == len(t):
            newS = sorted(s)
            newT = sorted(t)
            return newS == newT
        

File: 0242-valid-anagram/0242-valid-anagram.java
import java.util.*;
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length() == t.length()){
            char[] S = s.toCharArray();
            char[] T = t.toCharArray();
            Arrays.sort(S);
            Arrays.sort(T);
            return Arrays.equals(S,T);

        }
        return false;
    }
}